html/pfappserver/lib/pfappserver/Form/Config/Firewall_SSO/Checkpoint.pm

```diff
@@ -0,0 +1,134 @@
+package pfappserver::Form::Config::Firewall_SSO::Checkpoint;
+
+=head1 NAME
+
+pfappserver::Form::Config::Firewall_SSO::Checkpoint - Web form to add a Checkpoint firewall
+
+=head1 DESCRIPTION
+
+Form definition to create or update a Checkpoint firewall.
+
+=cut
+
+use HTML::FormHandler::Moose;
+extends 'pfappserver::Form::Config::Firewall_SSO';
+with 'pfappserver::Base::Form::Role::Help';
+
+use pf::config;
+use pf::util;
+use File::Find qw(find);
+
+## Definition
+has 'roles' => (is => 'ro', default => sub {[]});
+
+has_field 'id' =>
+  (
+   type => 'Text',
+   label => 'Hostname or IP Address',
+   required => 1,
+   messages => { required => 'Please specify the hostname or IP of the Checkpoint' },
```

Comment:
to be even more picky: Checkpoint -> Check Point

I would hate if someone would call our solution PaK eT Fen Ce